packages/plugin-lightningcss/src/types.ts

```diff
@@ -0,0 +1,38 @@
+import type { CustomAtRules, TransformOptions } from 'lightningcss';
+
+type Implementation = typeof import('lightningcss');
+
+export type LightningCssTransformOptions = Omit<
+  TransformOptions<CustomAtRules>,
+  'filename' | 'code' | 'inputSourceMap'
+>;
+
+export type LightningCssLoaderOptions = LightningCssTransformOptions & {
+  implementation?: Implementation;
```

Comment:
Can we move the `implementation` field to the root of `PluginLightningcssOptions` option and add document for this field.

Reply:
So we do not need two `implementation` field in both `transform` and `minify`.